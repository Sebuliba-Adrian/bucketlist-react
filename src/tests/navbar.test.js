import React from 'react';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-15';
import Navbar from '../components/navbar';

const props = {
  request: jest.fn(),
  viewBucketlists: jest.fn(),
  search: jest.fn(),
};

global.localStorage = {
  setItem: () => {},
  getItem: () => {},
};

it('renders correctly', () => {
  const navbar = renderer.create(<Navbar {...props} />).toJSON();
  expect(navbar).toMatchSnapshot();
});
