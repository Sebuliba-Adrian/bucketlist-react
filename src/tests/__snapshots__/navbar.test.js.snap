// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=""
>
  <div
    className="row"
  >
    <div
      className="col-md-4"
    >
      <a
        className="navbar-brand logo text-white"
        href=""
        onClick={[Function]}
      >
        <h3>
          <span
            className="fa fa-lg fa-bitbucket mr-3 ml-3"
          />
          |   Bucketlists
        </h3>
      </a>
    </div>
    <div
      className="col-md-4 text-center"
    >
      <input
        className="form-control mr-sm-2 search-box"
        onKeyDown={[Function]}
        placeholder="Search e.g b:the bucket title or i:the item title"
        type="text"
      />
    </div>
    <div
      className="col-md-4 text-right"
    >
      <div
        className="dropdown logout-dropdown"
      >
        <div
          className="fa fa-user-circle fa-lg text-white dropdown-toggle"
          data-toggle="dropdown"
          id="dropdownMenuLink"
        >
           
          <span
            className="logo"
          >
            <b>
               
               
            </b>
          </span>
        </div>
        <div
          aria-labelledby="dropdownMenuLink"
          className="dropdown-menu dropdown-cust"
        >
          <a
            className="dropdown-item"
            href=""
            onClick={[Function]}
          >
            Logout
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
